components/style/values/generics/position.rs

```diff
@@ -294,7 +49,7 @@ impl<H: ToComputedValue, V: ToComputedValue> ToComputedValue for Position<H, V>
 
     #[inline]
     fn from_computed_value(computed: &Self::ComputedValue) -> Self {
-        Position {
+        Self {
```

Comment:
I'm fine if the params and return types use `Self`, but wouldn't this be confusing? I'll leave it to you.